app/views/jobs/show.html.erb

```diff
@@ -8,3 +8,14 @@ Level of Interest: <%= @job.level_of_interest %>
 <br>
 
 Company: <%= link_to @job.company.name, company_path(@job.company) %>
+
+<%= form_for @comment do |f| %>
+<%= f.label :comment %>
+<%= f.text_area :content %>
+<%= hidden_field(:job, :id) %>
```

Comment:
we talked about this in class - if you nest your resources here, we won't need to pass a hidden field 👍 

if you have questions about this, please let me know!!